src/OOUI/TranslationsFieldset.php

```diff
@@ -21,6 +23,7 @@ public function __construct(array $config = [])
         $currentFieldset = new FieldsetLayout();
         $fieldsets = [$currentFieldset];
         $prevTranslation = null;
+        $this->disabled = $config['disabled'] ?? false;
```

Comment:
'disabled' config is dealt with in Element.php, (FieldsetLayout -> Layout -> Element) so this shouldn't be needed.

Reply:
Doesn't seem to work for me.

Reply:
Meh, that's not good for upstream, but I'll take a look later in OOUI while we use this here.